### DISCUSSION TITLE
Vertipaq Analyzer Stuck

### DISCUSSION THREAD

**Discussion by SemanticLair** [DaxStudio/DaxStudio#731] [Q&A] [answered]
I have a composite model that uses aggregations and RLS. The Vertipaq Analyzer in DAX Studio just spins and says "Loading model metrics".  Is VA compatible with composite models and RLS? When I change all tables to IMPORT mode, Vertipaq Analyzer works fine.

**Comment by dgosbell** [comment, marked as answer]
> Is VA compatible with composite models and RLS?

RLS does not matter as you should be running as an author/admin to run VA and RLS is not applied to admins. It should work with composite models although depending on your data source it could be slow. Try going into File > Options > Advanced and turn off the **read statistics from data** option and see if that helps

**Reply by SemanticLair** [reply]
_"Try going into File > Options > Advanced and turn off the read statistics from data"_

That worked perfectly... thanks!!
